ui/segmented-bar.info/sample/main.reel/main.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * @module ui/main.reel
+ */
+var Component = require("ui/component").Component;
+
+/**
+ * @class Main
+ * @extends Component
+ */
+exports.Main = Component.specialize(/** @lends Main# */ {
+    // SegmentedBar1
+    currentSegment: {value: null},
+    handleSegmentedBar1Action: {
+        value: function (event) {
+            console.log("heard");
+            this.currentSegment = event.detail.get('data').region;
+        }
+    },
+
+    // SegmentedBar2/3
+    content: {value: [
+        {"region": "region1", "size": 10, "flag": true},
+        {"region": "region2", "size": 10, "flag": true},
+        {"region": "region3", "size": 10, "flag": true},
+        {"region": "region4", "size": 10, "flag": true},
+        {"region": "region5", "size": 10, "flag": true},
+        {"region": "region6", "size": 50, "flag": true}
+    ]},
+    tree: {value: {
+        region: "USA",
+        size: 99,
+        regions: [
+            {
+                region: "Canada",
+                size: 20
+            },
+            {
+                region: "Ivory Coast",
+                size: 30,
+                regions: [
+                    {
+                        region: "China",
+                        size: 40
+                    }
+                ]
+            }
+        ]
+    }}
+});
```

Comment:
See the comments for ui/segmented-bar.info/main.reel/main.js, they all apply here.